hr_timesheet_report_invoice/report/account_analytic_line_inherit.xml

```diff
@@ -1,10 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <template id="account_analytic_line_inherit" inherit_id="hr_timesheet.report_timesheet">
         <xpath expr="//t[@t-set='show_project']" position="after">
             <t
                 t-set="invoice_name"
-                t-value="docs.sudo().get_invoice_name()"
+                t-value="docs and docs[0].sudo().get_invoice_name()"
```

Comment:
And why this change?

Reply:
But in that case this should be enough:
```suggestion
                t-value="docs and docs.sudo().get_invoice_name()"
```
and we keep multiple `docs` selection. Notice that `get_invoice_name()` is specifically designed to be called with a set of timesheets, passing only one breaks the functionality (see definition).

Reply:
Review again, making a rollback in this line (keeping v15 original code) and still works, so no changes for this line should be made in v17 IMO, please remove it